services/meta/errors.go

```diff
@@ -55,6 +55,10 @@ var (
 	// with an existing policy.
 	ErrRetentionPolicyConflict = errors.New("retention policy conflicts with an existing policy")
 
+	// ErrRetentionPolicyDurationBelowShardDuration is returned when updating a retention
+	// policy that has a duration lower than the current shard duration.
+	ErrRetentionPolicyDurationBelowShardDuration = errors.New("retention policy duration must be greater than the shard duration")
```

Comment:
Hm, it is a really long name, but I think it matches with the naming convention of the other errors. I don't think `ErrShortDuration` says enough about the error since it can also applies to `ErrRetentionPolicyDurationTooLow`.